README.md

```diff
@@ -2,10 +2,86 @@
 
 [![Build Status](https://travis-ci.org/manifestival/manifestival.svg?branch=master)](https://travis-ci.org/manifestival/manifestival)
 
-Manipulate unstructured Kubernetes resources loaded from a manifest
+Manipulate unstructured Kubernetes resources loaded from a manifest.
+
+Manifestival helps building operators by providing mechanisms to create/apply/delete resources from manifests.
+That means, you can feed your YAML to manifestival in your operator similar to feeding your YAML to kubectl, but on runtime.
+
+Manifestival also provides transform functionality to shape your resources before pushing them to Kubernetes. 
 
 ## Usage
 
+Basic Usage
+
+Create a manifest from YAML files in a directory:
+
+```go
+recursive  := True // or False
+restConfig := getMyClientGoRestConfig() 
+
+manifest, err := mf.NewManifest("/path/to/resources-dir", recursive, cfg)
```

Comment:
We've changed the way options are passed as parameters, so we'll need to change this slightly.